src/ch/imedias/rsccfx/view/RsccRequestPresenter.java

```diff
@@ -100,4 +153,87 @@ private void initHeader() {
     headerPresenter.setSettingsBtnAction(event ->
         popOverHelper.settingsPopOver.show(view.headerView.settingsBtn));
   }
+
+  /**
+   * Calls createSupporterList() and creates a button for every supporter found.
+   */
+
+  private void initSupporterList() {
+    createSupporterList();
+    for (int counter = 0; counter < supportAddresses.size(); counter++) {
+      createNewSupporterBtn();
+      setTextOnButtons(counter);
+    }
+    createNewSupporterBtn();
+  }
```

Comment:
This seems like it is somehow duplicated code with method "createSupporterList();".
@martinfrancois : please have a look